openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml

```
@@ -1,5 +1,8 @@
 type: object
-description: Priority settings.
+description: |-
+  Priority level settings for AML source matches. \
+  `p0` is critical priority level, and `p3` is low priority level. \
+  All source types in all hits are considered, and the one with the highest priority level is set in the resulting AML check.
```

suggestion: minor suggestion
```suggestion
  All source types are assessed when AML matches are found. 
  Sources with the highest priority levels are added to the AML check result.
```

---

This is incorrect though. The highest priority overrides the others.

---

I've removed it from this area and changed description of the field in AmlCheck.